Examples/js/main.js

```diff
@@ -120,7 +140,7 @@ function opencvIsReady() {
   }
   info.innerHTML = '';
   initUI();
-  slam = new cv.SLAM('ORBvoc.txt', 'c270.yaml', cv.MONOCULAR)
+  slam = new cv.SLAM('ORBvoc.bin', 'c270.yaml', cv.MONOCULAR);
```

Comment:
It still uses `cv` instead of `Module`. Does it work?

Reply:
It's my mistake. I forgot to replace the sample code.